src/main/scala/viper/gobra/util/GobraExecutionContext.scala

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+
+package viper.gobra.util
+
+import scala.concurrent.ExecutionContext
+
+trait GobraExecutionContext extends ExecutionContext
```

Comment:
Why not put `GobraExecutionContext` and `DefaultGobraExecutionContext` in the same file?

Reply:
Great idea! I think I'm still thinking the Java way, i.e. having one file per "thing"